TrafficCapture/trafficCaptureProxyServerTest/build.gradle

```diff
@@ -40,38 +57,66 @@ dependencies {
     }
 }
 
+CommonConfigurations.applyCommonConfigurations(project)
 configurations {
     all {
-        resolutionStrategy.dependencySubstitution {
-            substitute module('org.apache.xmlgraphics:batik-codec') using module('org.apache.xmlgraphics:batik-all:1.15')
-        }
+        exclude group: 'org.apache.logging.log4j', module: 'log4j-slf4j-impl'
```

Comment:
What do you need this line for?  What are you excluding it from?

Reply:
I needed to use it to prevent classpath issues caused by multiple logging implementations, which was occuring due to mulptiple dependencies pulling in this Log4j to SLF4J bridge as a transitive dependency.
I would get this exception without it: `Exception in thread "main" java.lang.NoSuchMethodError: 'void org.apache.logging.slf4j.Log4jLoggerFactory.<init>(org.apache.logging.slf4j.Log4jMarkerFactory)'`

Reply:
If I understand correctly, JMeter and the ProxyCapture project are both bringing in log4j components?  Do you know what version you're fixing on?